crates/uv-interpreter/src/python_query.rs

```diff
@@ -524,15 +540,56 @@ mod windows {
             return false;
         }
 
-        // Ex) `Local`
-        if !components
-            .next()
-            .is_some_and(|component| component.as_os_str() == "Local")
-        {
+        // The file is only relevant if it's a reparse point.
+        let md = match std::fs::symlink_metadata(path) {
+            Ok(md) => md,
+            Err(_) => return false,
+        };
+        if md.file_attributes() & FILE_ATTRIBUTE_REPARSE_POINT == 0 {
             return false;
         }
 
-        true
+        let mut path_encoded = path
+            .as_os_str()
+            .encode_wide()
+            .chain(std::iter::once(0))
```

Comment:
Previously, this wouldn't work for one of the two shims. But adding the null terminator seems to fix it on my machine.

Reply:
Adding the null termination seems correct. OsStr is not null terminated https://doc.rust-lang.org/std/ffi/struct.OsString.html